google-cloud-storage/src/main/java/com/google/cloud/storage/GrpcStorageImpl.java

```diff
@@ -1511,50 +1596,50 @@ public Iterable<ModelT> getValues() {
               .map(translator::decode)
               .iterator();
     }
+  }
 
-    private static <T> Stream<T> streamIterate(
-        T seed, Predicate<? super T> shouldComputeNext, UnaryOperator<T> computeNext) {
-      requireNonNull(seed, "seed must be non null");
-      requireNonNull(shouldComputeNext, "shouldComputeNext must be non null");
-      requireNonNull(computeNext, "computeNext must be non null");
-      Spliterator<T> spliterator =
-          new AbstractSpliterator<T>(Long.MAX_VALUE, 0) {
-            T prev;
-            boolean started = false;
-            boolean done = false;
-
-            @Override
-            public boolean tryAdvance(Consumer<? super T> action) {
-              // if we haven't started, emit our seed and return
-              if (!started) {
-                started = true;
-                action.accept(seed);
-                prev = seed;
+  private static <T> Stream<T> streamIterate(
```

Comment:
No actual code change to this helper method, simply un-nested it from `TransformingPageDecorator`